bin/bootstrap/ubuntu/14.04.sh

```diff
@@ -145,6 +153,13 @@ sudo mv nginx.conf /etc/nginx/nginx.conf
 # Reload with the new config
 sudo service nginx reload
 
+# Initialize MongoDB
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
+sudo apt-get update
+sudo apt-get install -y mongodb-org
+sudo service mongod status
```

Comment:
I use mongo because I was storing both the username/password combination and the username/token combinations in the same redis database. Plus, passportjs suggested that I use mongodb for my authentication system